### ISSUE TITLE
Adjust size/scale on hud

### ISSUE THREAD

**Issue by AzaleaFlame59** [Extegral/Universal-Clock-HUD#1]
**Provide a detailed and clear description of feature or change that you want to offer.**
Add a config option for something like a scale or sizing number, so the clock & time can be made bigger or smaller.

**Describe why you want this. If there's a reason.**
it's too big for me lol. It's a really nice hud doodad but it takes up just a bit more space than I'd really like it to; I think it'd be perfect otherwise.

**Comment by Aizistral** [comment]
Perhaps, but no promises. It's not very easy to resize rendered items in GUI actually.

**Comment by AzaleaFlame59** [comment]
Fair enough, if you can't do it I understand, and either way, thank you :D
